vllm/model_executor/layers/fused_moe/cutlass_moe.py

```diff
@@ -178,3 +179,119 @@ def cutlass_moe_fp8(
     if not apply_router_weight_on_input:
         c2 = c2 * topk_weights.view(m, topk, 1).to(out_dtype)
     return c2.sum(dim=1)
+
+
+FLOAT4_E2M1_MAX = scalar_types.float4_e2m1fn.max()
+FLOAT8_E4M3_MAX = torch.finfo(torch.float8_e4m3fn).max
+MAX_TOKENS_PER_EXPERT = 65536
```

Comment:
Why is this? Do we use `uint16_t` to represent the number of tokens per expert somewhere?

Reply:
For graph mode, I am allocating a statically sized large blockscale tensor for the activations quantization. I use this `m` to compute the size for blockscale tensor. The purpose is just a random big number that doesn't cause OOM.